ci/build/build-vscode.sh

```diff
@@ -47,6 +47,7 @@ main() {
   # as it is a submodule.
   export VSCODE_DISTRO_COMMIT
   VSCODE_DISTRO_COMMIT=$(git rev-parse HEAD)
+  VERSION=$(jq -r .version ../../package.json)
```

Comment:
Will this not be set by the workflow?

Reply:
AFAICT, we only build Code as part of the "Build code-server" job, which only runs in `build.yaml` which runs automatically so we have to grab it from the `package.json` here.

Is that right? Is there a better way to do this?

Reply:
Ahhh I saw `VERSION: "0.0.0"` and assumed that was for the build job but it was for the npm publish job.  I would be down to set `VERSION: "0.0.0"` for the build job as well.  That way the version is always set the same way (always by env var from a workflow).

Reply:
Good idea. Okay I'll do `VERSION: "0.0.0"`